contrib/slurm/slurmexecutor.py

```diff
@@ -313,10 +408,12 @@ def parse_seff(result):
     cpu_time_match = cpu_time_pattern.search(result)
     wall_time_match = wall_time_pattern.search(result)
     memory_match = memory_pattern.search(result)
+    exit_code = None
     if exit_code_match:
-        exit_code = int(exit_code_match.group(1))
+        status = str(exit_code_match.group(1))
+        exit_code = int(exit_code_match.group(2))
     else:
-        raise Exception(f"Exit code not matched in output: {result}")
+        status = "ERROR"
```

Comment:
In which situation does this happen? `ERROR` would be interpreted as a generic failure of the executed tool. If the problem is with something else, we should set a termination reason instead.

Reply:
`ERROR` will be translated to `failed` terminationreason, which I think is correct as a status here.

Reply:
Here, the status is still the status according to SLURM, not to Benchexec. That will be translated later, here: https://github.com/leventeBajczi/benchexec/blob/1895c8e09ff58714408f99f3196f933b5f41fc7a/contrib/slurm/slurmexecutor.py#L365-L372